src/MCPServer/install/README.md

```diff
@@ -153,11 +155,18 @@ This is the full list of variables supported by MCPServer:
     - **Type:** `int`
     - **Default:** `""`
 
-- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCHDIRECTORIESPOLLINTERVAL`**:
-    - **Description:** time in seconds between filesystem poll intervals.
-    - **Config file example:** `MCPServer.watchDirectoriesPollInterval`
-    - **Type:** `int`
-    - **Default:** `"1"`
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCH_DIRECTORY_METHOD`**:
+    - **Description:** how watched directory polling is done (`poll` or `inotify`). `inotify` is much more efficient, but only available
+    when using a local filesystem on Linux.
+    - **Config file example:** `MCPServer.watch_directory_method`
+    - **Type:** `string`
+    - **Default:** `"poll"`
+
+- **`ARCHIVEMATICA_MCPSERVER_MCPSERVER_WATCH_DIRECTORY_INTERVAL`**:
+     - **Description:** time in seconds between filesystem poll intervals.
+     - **Config file example:** `MCPServer.watch_directory_interval`
+     - **Type:** `int`
+     - **Default:** `"1"`
```

Comment:
We have a mechanism in place to handle backward-incompatible changes that you may want to use: https://github.com/artefactual/archivematica/blob/862c4cdd780f1a235f20716d8472d34d9b56094f/src/MCPClient/lib/settings/common.py#L81.

Reply:
Added a fallback to the old value in 1eccc979841297f2e635b652a10f144aa3fe384f.